channels/applicationinsights-channel-js/src/Sender.ts

```diff
@@ -670,29 +683,65 @@ export class Sender extends BaseTelemetryPlugin implements IChannelControls {
                 try {
                     let onCompleteFuncs = {
                         xdrOnComplete: (xdr: IXDomainRequest, oncomplete: OnCompleteCallback,payload?: IPayloadData) => {
-                            let data = _getPayloadArr(payload);
-                            if (!data) {
+                            let payloadArr = _getPayloadArr(payload);
+                            if (!payloadArr) {
                                 return;
                             }
-                            return _xdrOnLoad(xdr, data);
+                            const responseText = _getResponseText(xdr);
+                            let statsbeat = _core.getStatsBeat();
+                            var endpointHost = urlParseUrl(_self._senderConfig.endpointUrl).hostname;
+                            if (xdr && (responseText + "" === "200" || responseText === "")) {
+                                _consecutiveErrors = 0;
+                                if (statsbeat) {
+                                    statsbeat.count(200, payload, endpointHost);
+                                }
+                            } else {
+                                const results = parseResponse(responseText);
+                    
+                                if (results && results.itemsReceived && results.itemsReceived > results.itemsAccepted
+                                    && !_isRetryDisabled) {
+                                    if (statsbeat) {
+                                        statsbeat.count(201, payload, endpointHost);
+                                    }
+                                } else {
+                                    statsbeat.count(201, payload, endpointHost);
```

Comment:
What is a 201 status code? It doesn't seem to be a valid code from [breeze](https://github.com/aep-health-and-standards/Telemetry-Collection-Spec/blob/main/ApplicationInsights/sdk_behavior_breeze.md#the-response-returned-is-valid)